p2p/peer.py

```diff
@@ -17,9 +18,10 @@
     Generator,
     Iterator,
     List,
+    Sequence,
+    TYPE_CHECKING,
```

Comment:
nitpick, this seems to screw up alphabetical ordering

Reply:
I think `isort` which is implemented in one of our repos (maybe `eth-account`?) is the programatic solution for this.